modules/pam_pwhistory/opasswd.c

```diff
@@ -129,6 +129,7 @@ compare_password(const char *newpass, const char *oldpass)
 #ifdef HAVE_CRYPT_R
   struct crypt_data output;
 
+  memset(&output, 0, sizeof(struct crypt_data));
   output.initialized = 0;
```

Comment:
Setting `initialized = 0` at this and the other occurrences is not needed anymore.

Reply:
Dropped the unnecessary explicit assignments of `->initialized`.